### DISCUSSION TITLE
multi k8s use external mode to connect ceph cluster,if ceph cluster is down ,I backup and restored another ceph cluster,how did the cluster connectd to old ceph cluster change to connect to new ceph cluster

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13599] [Q&A] [answered]
multi k8s use external mode to connect ceph cluster,if ceph cluster is down ,I backup and restored another ceph cluster,how did the cluster connectd to old ceph cluster change to connect to new ceph cluster

**Comment by travisn** [comment]
Please provide more detailed repro steps, I don't understand the question.

**Reply by parth-gr** [reply]
@13567436138 do you mean how you can connect your standalone external new cephcluster back to the k8s cluster?
You can follow this document.https://rook.io/docs/rook/latest-release/CRDs/Cluster/external-cluster/
But you have to create a new pvc.

**Reply by 13567436138** [reply, marked as answer]
I mean rook use external mode to connected centerized rook。but the centerized rook is down，and I use backup tools,to backup centerized rook,it become a new centerized rook.how the external rook change to fit this situation.

**Reply by parth-gr** [reply]
@13567436138 that's a great question, you may need to reconnect to the new centralized rook in that case 

And would you provide some use cases of why we need to connect rook external to centralised rook, if we can have it by namespace separation

**Reply by 13567436138** [reply]
Mainly for failover
